PlugIns/GLSLang/CMakeLists.txt

```diff
@@ -19,7 +19,13 @@ elseif(DEFINED ENV{VULKAN_SDK})
     target_link_directories(Plugin_GLSLangProgramManager PUBLIC $ENV{VULKAN_SDK}/lib)
     target_link_libraries(Plugin_GLSLangProgramManager PUBLIC OgreMain shaderc_combined)
 else()
+  find_library(HLSL HLSL)
+  find_library(OGLCompiler OGLCompiler)
+  if(HLSL AND OGLCompiler)
     set(GLSLANG_LIBS glslang HLSL OSDependent OGLCompiler SPIRV SPIRV-Tools-opt SPIRV-Tools)
+  else()
+    set(GLSLANG_LIBS glslang OSDependent SPIRV SPIRV-Tools-opt SPIRV-Tools)
```

Comment:
please use the same pattern as  below for `GLSLANG_MI` instead of copy pasting the whole line